boards/arm64/mimx93_evk/mimx93_evk_a55.dts

```diff
@@ -36,4 +36,7 @@
 	/* clocks = <&ccm IMX_CCM_UART4_CLK 0x6c 24>; */
 	pinctrl-0 = <&uart2_default>;
 	pinctrl-names = "default";
+	assigned-clocks = <IMX93_CCM_LPUART2_ROOT>;
+	assigned-clock-parents = <IMX93_CCM_OSC_24M>;
```

Comment:
The deviation from the `clocks = <xx>` standard property here concerns me. I think we should still be using the `clocks` property for clock data, unless there is a good reason to deviate from the standard property nameused by other SOCs

Reply:
The problem with the `clocks` property is that it doesn't offer enough flexibility. For instance, the `assigned-clock-parents` property is optional (unless a node needs this property in which case it becomes mandatory). If we were to use the `clocks` then it would be mandatory to specify a clock's parent as the number of elements in the resulting clocks array would have to be divisible by some value. Also, the drivers don't need to have access to all of the fields (e.g: the clock's frequency or the clock's parent). A handle to their clock is the only thing they need.